scripts/screeps_tencent_batch_rl_runner.py

```
@@ -2064,6 +2079,150 @@ def verified_remote_policy_update_fields(
+    scope = required_non_empty_text(raw.get("candidateParameterScope"), "runtimeParameterInjection.candidateParameterScope")
+    injected_count = required_non_negative_int(
+        raw.get("injectedVariantCount"),
+        "runtimeParameterInjection.injectedVariantCount",
+    )
+    if "inlineCandidatesRuntimeInjected" in raw:
+        required_bool(raw.get("inlineCandidatesRuntimeInjected"), "runtimeParameterInjection.inlineCandidatesRuntimeInjected")
+    if "variantCount" in raw:
+        required_non_negative_int(raw.get("variantCount"), "runtimeParameterInjection.variantCount")
+    if runtime_injected:
+        if status != "injected":
+            raise BatchRunError("remote runtimeParameterInjection status must be injected when runtime injection is proven")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)